docs/new-audits.md

```diff
@@ -8,13 +8,33 @@ Lighthouse audits that surface in the report should be:
 - Contribute significantly towards making the mobile web experience better for end users. 
 - Not have a significant impact on our runtime performance or bundle size. 
 - Something that is new, and not something that is already measured by existing audits. 
-- Important for our strategic goals as a product.
-- Measurable (especially for performance audits) or have clear pass/fail states. 
+- Measurable (especially for performance audits) or have clear pass/fail states.
+- Actionable - when failing, specific advice should be given. If the failure can be tied to a specific resource (a DOM element, script, line of code), use the [appropriate detail type](https://github.com/GoogleChrome/lighthouse/blob/master/types/audit-details.d.ts). If multiple failures can occur for a page, return a table.
 - Not use 3rd party APIs for completing the audit check. 
 
+## Actionable
+
+1. When failing, specific advice should be given. If an audit can fail in multiple ways, each way should have a specific error message.
```

Comment:
i dont think its an "error message" because that indicates a runtime problem with the audit.

i dont know what the right words are but.. "users should understand the specifics of why this case is failing"